### ISSUE TITLE
octavia is upgraded out of order

### ISSUE THREAD

**Issue by samuelallan72** [canonical/charmed-openstack-upgrader#554]
See https://docs.openstack.org/charm-guide/latest/admin/upgrades/openstack.html#list-the-upgrade-order - octavia must be upgraded at the end.  However, COU places it in the 'control-plane' group, and upgrades it before nova-compute.

**Comment by gabrielcocenza** [comment]
We talked with the [OpenStack](https://chat.canonical.com/canonical/pl/zkbuci6g3tdqmftsknhepfw3cr) team and they seem to OK with upgrading octavia and openstack-dashboards out of the order from the docs.

Before changing this behavior I would ask ManSol if they want to change this. It might be easier for a management perspective to have a maintenance window for upgrading things that are not suppose to affect the running VMs and other that it's necessary to make VMs migrations. That is why we separated into control-plane and data-plane applications
